os/arch/arm/src/tiva/tiva_userspace.c

```diff
@@ -74,6 +74,14 @@
  * Private Data
  ****************************************************************************/
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
```

Comment:
Except boards we ported, other boards which come from Nuttx have the same format. We need to think which is better.